Cargo.toml

```diff
@@ -23,6 +23,7 @@ include = [
 redis = "^0.5"
 rustc-serialize = "^0.3"
 iron = "^0.2"
+hyper = "^0.7"
```

Comment:
I think not :( If you find a way to do this, let me know.

Btw, I think iron should pub use the necessary hyper types anyways. Unfortunately they don't right now. I created an issue.